class.ext_update.php

```diff
@@ -43,6 +56,10 @@ public function main()
 		ob_start();
 		$exectimeStart = microtime( true );
 
+		if ( $this->checkEnvironment() === false ) {
```

Comment:
It's easier to return string|null instead of using a class variable as workaround

Reply:
Done